src/client/src/error.rs

```diff
@@ -68,6 +69,13 @@ pub enum Error {
     /// Error deserialized from gRPC metadata
     #[snafu(display("{}", msg))]
     ExternalError { code: StatusCode, msg: String },
+
+    // Server error carried in Tonic Status's metadata.
+    #[snafu(display("{}", msg))]
```

Comment:
Do we need to print the status code? I found that ExternalError also only prints msg.